### DISCUSSION TITLE
[🧐] onClick or onclick, that's a question

### DISCUSSION THREAD

**Discussion by IanDxSSXX** [dlight-js/dlight#50] [Polls]
👋🏻 Hi! This is a poll on events' naming!

In DLight, setting a event is like this:
```js
button("click")
  .onclick(() => {
    console.log("CLICKED!")
  })
```
And it basically does:
```js
element.addEventListener("click", () => {
   console.log("CLICKED!")
})
```
And we're seeing a lot of other frameworks using small camel cases for events' naming conventions like `onClick` or `onMouseOver`. But we've got all lower cased event properties in vanilla js like `element.onclick = clickMe`.

It'll both be the same for dlight implementations. So I guess I need your help here. Just vote for your favourite!

**Comment by liaobinbin** [comment]
try  
```javascript
function handleClick(e){
  e.target.off.click(handleClick);
  console.log("CLICKED ONECE");
}

button("click")
  .on.click(handleClick)
````

Don't mind, I just wrote randomly.
Maybe I just simply don't want too many options to appear in my code completion tool when I type `.`

**Reply by IanDxSSXX** [reply]
Seems good. I'll add it to the options
